planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -2965,7 +2971,22 @@ TurnSignalInfo AvoidanceModule::calcTurnSignalInfo(const ShiftedPath & path) con
       turn_signal_info.turn_signal.command = TurnIndicatorsCommand::ENABLE_RIGHT;
     }
   } else {
-    turn_signal_info.turn_signal.command = TurnIndicatorsCommand::DISABLE;
+    for (const auto & cl : *debug_data_.current_lanelets) {
+      const auto neigbor_lane = planner_data_->route_handler->getAllSharedLineStringLanelets(cl);
```

Comment:
In my understanding, it needs only shift side adjacent/opposite lanelets, so I think it is better to use other function (e.g. `getRightLanelet`, `getLeftLanelet`, ...). What do you think?

Reply:
There are double-lane roads in the same direction on our map. I know these functions could be useful if all the double lanes have opposite directions. So that's why I use `getAllSharedLineStringLanelets` function.